packages/transit-vehicle-overlay/src/components/markers/ModeCircles/index.js

```diff
@@ -25,3 +25,7 @@ export const CircledVehicle = utils.makeRotatedMarker(
     );
   }
 );
+
+// Rename styled components for export
+const Styled = StyledCircle;
```

Comment:
I would `import * as Styled` and remove this statement and comment above.

Reply:
I think Landon's idea here was to prevent confusion of what `Styled.Shape` means... I've updated the export style however to avoid the variable definition